hpobench/benchmarks/ml_mmfb/tabular_benchmark.py

```diff
@@ -0,0 +1,230 @@
+from pathlib import Path
+from typing import Union, List, Dict
+
+import ConfigSpace as CS
+import numpy as np
+from ConfigSpace.read_and_write import json as json_cs
+
+from hpobench.abstract_benchmark import AbstractBenchmark
+from hpobench.dependencies.ml.ml_benchmark_template import metrics
+from hpobench.util.data_manager import TabularDataManager
+
+
+class BaseTabularBenchmark(AbstractBenchmark):
```

Comment:
Do we really need a Base Class here?
I'll add more reasons of why I think this is not required in the comment below.